torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,41 @@
+import io
+
+import torch
+
+from typing import Union
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
+    *args,
+    opset_version: Optional[int] = None,
```

Comment:
Why not `opset_version: int = <default version>,`?

Reply:
An explicit version as a default implies that version is part of the API contract. If today we say `opset_version: int = 14` and then decide that no, now `17` is the default, we have broken the expectation. By making `opset_version` optional with `None` as the default, we are free to change the default version over time.

Reply:
Hmm so I guess the idea is we explicit call out that the version is "undefined", when `opset_version` is `None`. So no consistent version should be expected. And the version can only be known by inspecting the exported model/log, but not from API.

I was under the impression that we would do something like below, which 'implicitly' makes the `default version` part of API contract.
```
def dynamo_export(...):
    """
    ...
        opset_version: ..., if None, defaults to xxx.
    """
```

This makes me wonder if we should set `opset_version` as required ... Because users are often not aware that they actually do care about opset version until they hit a problem.

Reply:
Removed top level options. `None` is default value